### DISCUSSION TITLE
convert lines

### DISCUSSION THREAD

**Discussion by southerndream56** [iptv-org/community#1003] [Q&A]
Hello ,
Under the Venezuelan group of channels there a few channels that have lines like this one :
#EXTINF:-1 tvg-id="Vive.ve" tvg-logo="https://i.imgur.com/VEYhkDq.png" group-title="Education",Vive (1080p) [Not 24/7]
#EXTVLCOPT:http-referrer=https://odysee.com/
https://cloud.odysee.live/content/fed20cff907550b0c7d83991fb3630e2a9f90454/master.m3u8

Does anybody know how to turn it into normal 2 lines definition so that it would be possible to watch it on Smart TV or Firestick ?
Great thanks

**Comment by freearhey** [comment]
In general, it should not affect the playback, but if it is really necessary, just delete all lines that start with `#EXTVLCOPT`.
